ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -803,7 +808,29 @@ public static boolean isPrivate(Map<String, Object> args) throws RestServiceExce
    * @throws RestServiceException if exception occurs during parsing the arg.
    */
   public static boolean isChunkUpload(Map<String, Object> args) throws RestServiceException {
-    return getBooleanHeader(args, Headers.CHUNK_UPLOAD, false);
+    return getBooleanHeader(args, Headers.CHUNK_UPLOAD, false, false);
+  }
+
+  /**
+   * Determine if {@link Headers#NAMED_UPSERT} is set in the request args.
+   * @param args The request arguments.
+   * @return {@code false} if {@link Headers#NAMED_UPSERT} is set to false else return true.
+   * @throws RestServiceException if exception occurs during parsing the arg.
+   */
+  public static boolean isUpsertForNamedBlob(final Map<String, Object> args) throws RestServiceException {
```

Comment:
private

Reply:
or get rid of it since there is only 1 usage

Reply:
It's created per your previous comment: https://github.com/linkedin/ambry/pull/2247#discussion_r1017462532
Since it was used in two different scripts, it has to to public?

Reply:
Ok, i removed it in latest commit.